apps/web/src/components/ExperienceAccordion/ExperienceAccordionContentSection.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { HeadingRank } from "@gc-digital-talent/ui";
+
+export interface ExperienceAccordionContentSectionProps {
+  title: string;
+  children: React.ReactNode;
+  headingLevel?: HeadingRank;
+}
+
+export const ExperienceAccordionContentSection = ({
```

Comment:
I think we should avoid mixing named and default exports, preferring default exports for single components.

Reply:
Good call.  Fixed in [Remove extra exports](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6457/commits/5dbaec08a9fc948b482c1f9e26765246d65cf2ed).